src/Sarif.Viewer.VisualStudio/ErrorListService.cs

```diff
@@ -223,6 +225,33 @@ private void WriteRunToErrorList(Run runLog)
             }
         }
 
+        private IEnumerable<IEnumerable<AnnotatedCodeLocation>> CreateAnnotationsFromStacks(IList<Stack> stacks)
```

Comment:
Note, the viewer is currently broken for results that don't populate the Location object (which is permitted by spec). That fix coming in a second change.